dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -111,6 +114,10 @@ def allows_silent_takeover(oauth_user)
     allow_takeover &= %w(facebook google_oauth2 windowslive).include?(oauth_user.provider)
     # allow_takeover &= oauth_user.email_verified # TODO (eric) - set up and test for different providers
     lookup_user = User.find_by_email_or_hashed_email(oauth_user.email)
-    allow_takeover && lookup_user && lookup_user.provider != 'clever' # do *not* allow silent takeover of Clever accounts!
+    if cookies['pm'] == 'clever_takeover'
+      allow_takeover && lookup_user
```

Comment:
Is it possible for `omniauth_callbacks_controller_test` to enable the pagemode and test this path?  Otherwise coverage in this controller is fairly good.

Reply:
I have had a lot of trouble getting that to work (the cookie wasn't actually getting set), it's an item on my eventual TODO list to make testing stuff behind pagemode flags more reasonable; I'll definitely be adding tests for this when I pull this out from behind the flag